packages/connect-explorer/src/pages/changelog.mdx

```diff
@@ -52,4 +67,16 @@ export const Changelog = () => {
 
 };
 
+<VersionOverview />
+
+<CollapsibleBox heading="What is a Canary Release?" margin={{ top: spacings.md }}>
+    Canary releases are beta versions available before the stable release. They provide early access to new features, improvements, and (hot)fixes, allowing developers to test their applications with the latest changes. Unlike stable versions, which are extensively tested and released less frequently, canary versions are released more often to address immediate needs, such as critical fixes or compatibility updates required by specific third-party applications or implementations.
+
+    ### Why Use Canary Releases?
+    *Immediate fixes*: If your application encounters a critical issue with the current stable release, a canary version might already have the necessary fix, allowing you to resolve problems more quickly.
+
+    *Feedback Opportunity*: Using canary releases allows you to provide feedback on new features and fixes, influencing the final stable release and ensuring it works smoothly with your application. If you encounter any issue, bug, or incompatibility introduced in a Canary release, report it by creating a new issue in our repository.
```

Comment:
Please add a link to "creating a new issue in our repository" -> https://github.com/trezor/trezor-suite/issues/new/choose